src/components/experiment-tracking/details/details.js

```diff
@@ -45,6 +70,19 @@ const Details = ({
     return null;
   }
 
+  if (showSingleRunLoader) {
+    return (
+      <div
+        className={classnames('kedro', 'details-mainframe', {
+          'details-mainframe--sidebar-visible': sidebarVisible,
+        })}
+      >
+        <RunMetadataLoader theme={theme} />;
+        <RunDatasetLoader theme={theme} />;
```

Comment:
```suggestion
        <RunMetadataLoader theme={theme} />
        <RunDatasetLoader theme={theme} />
```

Those semicolons are rendering.